Userland/Utilities/uniq.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2020, Matthew L. Curry <matthew.curry@gmail.com>
- *
```

Comment:
It would be better to remove this change from the commit where it was first made, rather than fixing it in a later commit.